opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -584,7 +584,12 @@ namespace Opm {
                 well->setPrevSurfaceRates(this->wellState(), this->prevWellState());
             }
 
-            well->wellTesting(ebosSimulator_, simulationTime, this->wellState(), this->groupState(), wellTestState(), deferred_logger);
+            try {
+                well->wellTesting(ebosSimulator_, simulationTime, this->wellState(), this->groupState(), wellTestState(), deferred_logger);
+            } catch (std::runtime_error& e) {
```

Comment:
Any logic_error thrown will be uncaught, maybe change to catch std::exception?

Reply:
Good point, will update!